src/policies/activityReport.test.js

```diff
@@ -198,4 +198,24 @@ describe('Activity Report policies', () => {
       });
     });
   });
+
+  describe('canDelete', () => {
+    it('is true for author of draft report', () => {
```

Comment:
The ticket says that the creator and admins should be able to delete. Do we know who are the admins? Is it the managers? Or am I thinking of something else.

Reply:
You're right, I totally missed that part of the issue. I don't know who an report admin is either. Asking in GSA slack.